bee-api/bee-rest-api/src/lib.rs

```diff
@@ -3,4 +3,6 @@
 
 // #![deny(missing_docs, warnings)]
 
+extern crate bee_message;
```

Comment:
Is this really necessary ?

Reply:
hmm didn't add it manually, let me check